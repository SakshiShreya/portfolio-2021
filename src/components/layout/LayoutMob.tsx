import NavContainer from "../navigation/navContainer/NavContainer";
import Toolbar from "../navigation/toolbar/Toolbar";
import styles from "./Layout.module.scss";

export interface LayoutMobProps {}

const LayoutMob: React.FC<LayoutMobProps> = () => {
  return (
    <div className={styles.main}>
      <Toolbar />
      <NavContainer />
    </div>
  );
};

export default LayoutMob;
