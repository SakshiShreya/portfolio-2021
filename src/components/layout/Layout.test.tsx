import { HashRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import LayoutDsk from "./LayoutDsk";
import LayoutMob from "./LayoutMob";

it("should render Desktop layout properly", () => {
  const elem = renderer
    .create(
      <HashRouter>
        <LayoutDsk />
      </HashRouter>
    )
    .toJSON();
  expect(elem).toMatchSnapshot();
});

it("should render Mobile layout properly", () => {
  const elem = renderer
    .create(
      <HashRouter>
        <LayoutMob />
      </HashRouter>
    )
    .toJSON();
  expect(elem).toMatchSnapshot();
});
