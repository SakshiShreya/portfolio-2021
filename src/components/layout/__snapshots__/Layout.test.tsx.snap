// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Desktop layout properly 1`] = `
<div
  className="main"
>
  <div
    className="cont"
  >
    <nav
      className="drawer "
    >
      <img
        alt="Logo"
        className="logo"
        src="/assets/logo/SSlogo.png"
      />
      <ul>
        <li
          className="li h4"
        >
          <a
            href="#/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="li h4"
        >
          <a
            href="#/about"
            onClick={[Function]}
          >
            About Me
          </a>
        </li>
        <li
          className="li h4"
        >
          <a
            href="#/experience"
            onClick={[Function]}
          >
            Experience
          </a>
        </li>
      </ul>
    </nav>
    <div
      className="cont slide"
      id="content"
    >
      <div
        className="cont"
      >
        <div>
          <p
            className="h1 head"
          >
            Hello World... 
          </p>
          <h2
            className="h1 head name"
          >
            I am 
            <span
              className="green"
            >
              S
            </span>
            akshi 
            <span
              className="orange"
            >
              S
            </span>
            hreya
            ,
          </h2>
          <p
            className="h1 head"
          >
            Web Developer
          </p>
          <p
            className="h2"
          >
            and Mentor.
          </p>
          <p
            className="psText"
          >
            I can develop responsive and pixel-perfect websites. I love what I do. I can mentor you to be the same.
          </p>
          <a
            className="btn secondary small border help"
            href="#/"
            onClick={[Function]}
          >
            How can I help you?
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`should render Mobile layout properly 1`] = `
<div
  className="main"
>
  <div
    className="toolbar"
  >
    <button
      className="hamburger "
      onClick={[Function]}
    >
      <svg
        height="40"
        viewBox="0 0 100 100"
        width="40"
      >
        <path
          className="line line1"
          d="M 20,29.000046 H 80.000231 C 80.000231,29.000046 94.498839,28.817352 94.532987,66.711331 94.543142,77.980673 90.966081,81.670246 85.259173,81.668997 79.552261,81.667751 75.000211,74.999942 75.000211,74.999942 L 25.000021,25.000058"
        />
        <path
          className="line line2"
          d="M 20,50 H 80"
        />
        <path
          className="line line3"
          d="M 20,70.999954 H 80.000231 C 80.000231,70.999954 94.498839,71.182648 94.532987,33.288669 94.543142,22.019327 90.966081,18.329754 85.259173,18.331003 79.552261,18.332249 75.000211,25.000058 75.000211,25.000058 L 25.000021,74.999942"
        />
      </svg>
    </button>
    <img
      alt="Logo"
      className="logo"
      height="100"
      src="/assets/logo/SSlogo.png"
    />
  </div>
  <div
    className="cont"
  >
    <nav
      className="drawer "
    >
      <img
        alt="Logo"
        className="logo"
        src="/assets/logo/SSlogo.png"
      />
      <ul>
        <li
          className="li h4"
        >
          <a
            href="#/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="li h4"
        >
          <a
            href="#/about"
            onClick={[Function]}
          >
            About Me
          </a>
        </li>
        <li
          className="li h4"
        >
          <a
            href="#/experience"
            onClick={[Function]}
          >
            Experience
          </a>
        </li>
      </ul>
    </nav>
    <div
      className="cont slide"
      id="content"
    >
      <div
        className="cont"
      >
        <div>
          <p
            className="h1 head"
          >
            Hello World... 
          </p>
          <h2
            className="h1 head name"
          >
            I am 
            <span
              className="green"
            >
              S
            </span>
            akshi 
            <span
              className="orange"
            >
              S
            </span>
            hreya
            ,
          </h2>
          <p
            className="h1 head"
          >
            Web Developer
          </p>
          <p
            className="h2"
          >
            and Mentor.
          </p>
          <p
            className="psText"
          >
            I can develop responsive and pixel-perfect websites. I love what I do. I can mentor you to be the same.
          </p>
          <a
            className="btn secondary small border help"
            href="#/"
            onClick={[Function]}
          >
            How can I help you?
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
